runtime/recipe/recipe-util.js

```diff
@@ -94,6 +93,11 @@ class RecipeUtil {
           } else if (forward.has(shapeVC.view) && forward.get(shapeVC.view) !== null) {
             continue;
           }
+          if ((shapeVC.view.fate != 'create' || (recipeVC.view.fate != 'create' && recipeVC.view.originalFate != 'create')) &&
```

Comment:
Could you add a comment to what where're checking?

Reply:
done